src/pages/ProductView/ProductView.js

```diff
@@ -1,7 +1,188 @@
-import React from 'react';
+import { React, useState, useEffect } from 'react';
```

Comment:
React는 named export가 아닌 default export입니다.
```suggestion
import React, { useState, useEffect } from 'react';
```